default/python/README.md

```diff
@@ -110,3 +82,75 @@ Hook will track config file and update himself in case of changes.
 Notes:
 - First commit after installation/update of hooks may take some time.
 - You can skip checks by adding `-n, --no-verify` to `git commit`.
+
+## Developers routine
+### Manual code style check
+You don't need to run this script manually, pre-commit hooks and CI will do it fo you.
+
+We use [flake8](https://pypi.python.org/pypi/flake8) for code style checks.
+Settings for flake8 located in the `tox.ini` across the project.
+
+Flake should be run from directory where `tox.ini` located:
+```sh
+cd default/python
+flake8
+```
+
+We use [black](https://pypi.org/project/black/) for code formatting.
+Settings for black are located in `pyproject.toml`.
```

Comment:
Perhaps isort and pytest could be mentioned as well. I constantly learn about new tests when one of my PRs fail the github hooks.

Reply:
Added